src/pretix/control/views/checkin.py

```diff
@@ -33,12 +34,18 @@ def get_queryset(self, filter=True):
             m=Max('datetime')
         ).values('m')
 
+        cqs2 = Checkin.objects.filter(
```

Comment:
We should try to eliminate this sub-query

Reply:
See below